### ISSUE TITLE
Use Makarios' approach to simplify Tarski's axioms of geometry

### ISSUE THREAD

**Issue by david-a-wheeler** [metamath/set.mm#976]
The paper "A further simplification of Tarski's axioms of geometry" by Timothy Makarios is at: https://arxiv.org/abs/1306.0066  .

It tweaks the axioms to allow "another of the axioms to be omitted  from the set of axioms and proven as a theorem."  In particular, by  tweaking one axiom, another axiom turns into a provable theorem.

In general we try to eliminate axioms, so it seems like a useful thing to (eventually) pursue.

**Comment by tirix** [comment]
Actually, this was issue #964.
I provided there the name of the axiom to tweak and the name of the axiom that can be removed, sorry if it was not clear! 😅

**Comment by benjub** [comment]
@tirix Not sure how you plan to do it, and whether this is a good idea or not.  Makarios's trick is to twist the conclusion of the five-segments axiom to "incorporate" the reflexivity axiom, thus getting rid of the latter.  This may obscure what is going on, plus the set of axioms involving only congruence is self-sufficient in the current presentation, but not with Makarios's trick.

**Comment by tirix** [comment]
Though I did not plan in detail, I thought about the obvious: remove the “reflexivity of congruence” part in the definition “df-trskgc”, and then re-prove “axtgcgrrflx”, from the other definitions, probably without changing the name, so that the rest is unchanged.

With the explanations in the comments of ~ axtgcgrreflx, including the reference to Makario, I believe people would still be able to follow what’s going on.

Until now, though, I have not started any of this.

**Comment by david-a-wheeler** [comment]
Maybe it's not worth it.  It does reduce the absolute number of axioms, but that's a somewhat naive way to count simplicity.  We could instead document that we know *about* it, and note that we haven't done it (perhaps with some discussion as to why).

**Comment by benjub** [comment]
I tend to agree with @david-a-wheeler here.  But @tirix is at the command, so I'll leave it to him, and once it is documented one way or another, this issue may be closed.
